lib/transforms/json-to-html-data.js

```diff
@@ -328,15 +328,15 @@ function getNotes (data) {
   }
 }
 
-function getObjectType (data) {
+function getObjectType (data, links) {
   var value;
   if (data.type !== 'people' && data.attributes.name) {
     data.attributes.name.forEach(el => {
       if (el.primary) {
         value = el.value || null;
       }
     });
-    return value ? {key: 'type', value: value} : null;
+    return value ? {key: 'type', value: value, link: links.root + '/search/objects?filter[type]=' + value} : null;
```

Comment:
`value` in URL needs `encodeURIComponent` in case it contains URI reserved characters like & for example.